src/class/SimpleData.ts

```diff
@@ -83,15 +81,36 @@ export default class SimpleData {
     get keys() {
         return this._keys
     }
-    set keys(data) {
-        this._keys = data[0] === undefined ? [] : Object.keys(data[0])
+    set keys(keys) {
+        this._keys = keys === undefined ? [] : keys
     }
 
     #updateSimpleData(data: SimpleDataItem[]) {
         this._data = data
         this._keys = data[0] === undefined ? [] : Object.keys(data[0])
     }
 
+    @logCall()
+    loadLocalFile({
+        path,
+        missingKeyValues = { "null": null, "NaN": NaN, "undefined": undefined },
```

Comment:
Should we avoid mutable objects as parameters input?
Not sure about this one (I know that's valid for Python, but I dont know about JS).

Reply:
Good point. I don't know! I guess for the moment it's okay like that.